_articles/de/getting-paid.md

```diff
@@ -1,7 +1,7 @@
 ---
 lang: de
-title: Getting Paid for Open Source Work
-description: Sustain your work in open source by getting financial support for your time or your project.
+title: Für Open-Source-Arbeit bezahlt werden
+description: Unterstützen Sie Ihre Arbeit in Open Source, indem Sie finanzielle Unterstützung für Ihre Zeit oder Ihr Projekt erhalten.
```

Comment:
So do we use formal forms as of now?

Reply:
Let's have this discussion in #577, not in [yet another diff](https://github.com/katrinleinweber/opensource.guide/commit/920ecc1efcacf2a4fdde16c720426cc567dcebe2#r28031116) ;-)